commands/hangman/index.ts

```diff
@@ -0,0 +1,205 @@
+import { Message, MessageEmbed } from "discord.js";
+import { GameState, MessageParams } from "./hmTypes";
+import dictionary from "./dictionary.json";
+import { HANGMANPICS } from "./asciiArts";
+
+const TOTALGUESSES = 6;
+
+const gameState: GameState = {
+    active: false,
+    guessedCharacters: [],
+    knownCharacters: [],
+    guessesLeft: TOTALGUESSES,
+    word: "",
+    wordExplanation: "",
+    errorMessage: "",
+};
+const messageParams: MessageParams = {
+    hangManMessage: undefined,
+    guessCharacter: "",
+    messageContent: "",
+    messageChannel: undefined,
+};
+
+/* MAIN */
+const hangMan = async () => {
```

Comment:
Hangman on yleensä yhdyssana, tää camelcase on väärin, pitäis olla `hangman`

Reply:
Sori, tässä yhteydessä funktion nimi derivoituu sanoista 'Hang' & 'Man'. Pelin tarkoituksena on siis hirttää mies eikä pelata mitään typerää lasten peliä.

Harkitsen asiaa.